autoload/vimteractive.vim

```diff
@@ -121,11 +121,18 @@ function! vimteractive#term_start(term_type)
 	" Create a new term
 	echom "Starting " . l:term_command
     let l:term_bufname = s:new_name(l:term_type)
-	call term_start(l:term_command, {
-		\ "term_name": l:term_bufname,
-		\ "term_finish": "close",
-		\ "term_kill": "term"
-		\ })
+	if v:version < 801
+        call term_start(l:term_command, {
+		    \ "term_name": l:term_bufname,
```

Comment:
Could we fix the tab indentation here so they're expanded into spaces?

Reply:
Done! I thought I had expandtab on...